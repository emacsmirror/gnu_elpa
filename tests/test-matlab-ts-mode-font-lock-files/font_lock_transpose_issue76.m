% -*- matlab-ts -*-

% https://github.com/acristoffers/tree-sitter-matlab/issues/76

someFcn((1:10)')
someFcn((1:10).')

n = 5;
coeff = char((0:2*n-1)' + 'a')
