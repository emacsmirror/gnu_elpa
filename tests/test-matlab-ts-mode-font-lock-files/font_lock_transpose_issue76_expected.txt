c ccc ccccccccc ccc

c cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc

dddddddbbnDnnbob
dddddddbbnDnnboob

v o nD
vvvvv o BBBBbbnDnodonbo o SsSb
